packages/cli/src/genChangelog.ts

```diff
@@ -0,0 +1,43 @@
+import {parseLogParameters} from "./consoleParamsReader";
+import {generateChangeLogFile} from "./changelogFileWriter";
+import {Config, exampleConfig} from "../../shared/type";
+import {validateFileNameSchema} from "./validators";
+import {isLeft} from "fp-ts/lib/Either";
+
+const extractFieldsFromFileName = (config: Config): string[] => {
+    const regex = /[${][^}]+[}]/g;
+    if (config.filename == null) {
+        return [];
+    }
+    const fieldsFoundRaw = config.filename.match(regex);
+    const fieldsFound: string[] = [];
+    if(fieldsFoundRaw == null) {
+        return fieldsFound;
+    }
+    fieldsFoundRaw.forEach((field) => {
+        fieldsFound.push(field.substring(2, field.length - 1));
+    });
+
+    return fieldsFound;
```

Comment:
Hier eine lösung ohne substring und einwenig eindeutigerem Regex:
```suggestion
    const regex = /\${(\w+)}/g;
    if (config.filename == null) {
        return [];
    }
    const result: string[] = [];
    for (let match = regex.exec(config.filename); match; match = regex.exec(config.filename)) {
        result.push(match[1]);
    }
    return result;
```